.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+dist: xenial
+sudo: false
+
+cache: pip
+
+install:
+  - pip install --upgrade pycodestyle
```

Comment:
Probably overkill for now, but we could just use `pip install -r requirements.txt`, which will also set up Travis for any future testing.